code/modules/flufftext/TextFilters.dm

```diff
@@ -152,15 +152,15 @@ proc/RadioChat(mob/living/user, message, distortion_chance = 60, distortion_spee
 					if("s")
 						newletter = "$"
 					if("e")
-						newletter = "�"
+						newletter = "�"
 					if("w")
-						newletter = "�"
+						newletter = "�"
 					if("y")
-						newletter = "�"
+						newletter = "�"
 					if("x")
-						newletter = "�"
+						newletter = "�"
 					if("u")
-						newletter = "�"
+						newletter = "�"
```

Comment:
Is this meant to be this kind of deep-fried Unicode? (yes, guerilla review, hello)